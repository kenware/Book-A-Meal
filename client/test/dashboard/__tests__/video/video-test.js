import React from 'react';
import { shallow } from 'enzyme';
import Video from '../../../../components/dashboard/video';

describe('Test Video component of dashboard', () => {
  it('renders correctly', () => {
    const tree = shallow(<Video />);
    expect(tree).toMatchSnapshot();
  });
  it('should have 1 video Class', () => {
    const tree = shallow(<Video />);
    expect(tree.find('.video')).toHaveLength(1);
  });
});
