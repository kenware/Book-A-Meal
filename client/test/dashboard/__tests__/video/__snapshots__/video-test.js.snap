// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Video component of dashboard renders correctly 1`] = `
<div
  className="video"
>
  <h3
    className="p-color text-center"
  >
    While waiting for today menu, you might want to watch this video 
  </h3>
  <h5
    className="text-center"
  >
     French Recipe
  </h5>
  <div
    className="video-container"
  >
    <Player
      aspectRatio="auto"
      fluid={true}
      muted={false}
      playsInline={false}
      poster="https://img.buzzfeed.com/thumbnailer-prod-us-east-1/13b12570812444f1bc86415b6e4c5284/BFV39875_VeganLunchMealPrep_FB_FINAL_FINAL_FINAL.jpg"
    >
      <source
        src="https://vid.buzzfeed.com/output/83972/landscape_720/1520642061"
      />
      <BigPlayButton
        position="center"
      />
    </Player>
  </div>
</div>
`;
